-- vim: set ft=rb:

module self
  def main()
    try
      throw("Error")
    catch(msg : String)
      trace("Error occurred: #{msg}")
    end
  end
end
